sources/packages/web/src/components/aest/institution/modals/ApproveDenyDesignationModal.vue

```diff
@@ -26,7 +26,7 @@
                 variant="outlined"
                 :rules="[checkStringDateFormatRule]"
                 hide-details="auto" /></v-col></v-row
-          ><v-divider></v-divider>
+          ><v-divider class="border-opacity-100" :thickness="2" />
```

Comment:
I had the same feeling towards thickness as well. Instead of setting it 2 everywhere is there a way that we can bring that from constant. I know that we stay away from using typescript constants on vue components. @ann-aot tried to check if that works by setting defaults for a component at vuetify.ts and it didn't work. 

This is not a blocker comment. but trying to see if anyone has a better suggestion .

@andrewsignori-aot @andrepestana-aot @guru-aot ?

Reply:
I think that should be set via CSS: 
```css
.v-divider {
  opacity: 1 !important;
  border-top: 2px solid $common-border-color !important;
}
```

Reply:
The styles can be used as @andrepestana-aot mentioned, I'm just not sure about the "border-top" for instance.
I wold look at the most specific style and override it.
For instance, probably vertical and horizontal may have different ones.